David-Lectamo/tarea-5/game.js

```diff
@@ -0,0 +1,235 @@
+const ordened = [
+  [1 ,2 ,3 ,4 ],
+  [5 ,6 ,7 ,8 ],
+  [9 ,10,11,12],
+  [13,14,15,  ],
+];
+
+const numbers = [
+  [13,4 ,15,7 ],
+  [14,1 ,9 ,11],
+  [6 ,8 ,10,3 ],
+  [5 ,2 ,12,  ],
+];
+
+function init(){
+  const cell00 = document.getElementById('cell00'); 
```

Comment:
@L3C74M0 you should build a matrix wi these constants

Reply:
@L3C74M0 , The issue still there, the main problem is that you're defining 16 constants then assigning events to each one, calling addEventListener 16 times.

```suggestion
  const cells = [];
  for(let i=0; i < 4 ; i++) {
   for(let j=0; j < 4; j++) {
    cells[i][j] = document.getElementById(`cell${i}${j}`); 
    cells[i][j].addEventListener('click', () => {
      click(i,j,'topRightCorner');
    })
   }
  }
```